### PR TITLE
feat: anthos modules remote fleet project

### PR DESCRIPTION
**Pull request by CPL-markus**
It should be possible to enable the ACM feature on a GKE cluster that has its fleet in a separate project.

Therefore an additonal variable `hub_project_id` was introduced to reference the fleet project id. If it is empty (default), the `project_id` variable will be used instead.

### CODE REVIEW

**Comment by CPL-markus** [comment]
@apeabody Sorry, messed up the old commit while trying to patch something in Git. Please check this (new) request again.

**Review by apeabody — CHANGES_REQUESTED** [review]
Thanks for the contribution @CPL-markus!

```
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; 
        	            	Error: Failed to retrieve project, pid: , err: project: required field is not set
        	            	
        	            	  with module.example.module.acm.google_gke_hub_feature.acm[0],
        	            	  on ../../../modules/acm/feature.tf line 17, in resource "google_gke_hub_feature" "acm":
        	            	  17: resource "google_gke_hub_feature" "acm" {
        	            	}
        	Test:       	TestSimpleZonal
```

**Review by CPL-markus — COMMENTED** [review]
My bad, the evaluation of empty strings seems to require `coalesce`, instead of `try` to ensure the right variable is used.

File: modules/acm/variables.tf [hunk 1]
```diff
@@ -24,6 +24,12 @@ variable "project_id" {
   type        = string
 }
 
+variable "hub_project_id" {
```
Comment (apeabody) [inline modules/acm/variables.tf:27]: I think `fleet` might be more clear for the usage, match `enable_fleet_registration`, etc.

```suggestion
variable "fleet_project_id" {
```

**Review by apeabody — CHANGES_REQUESTED** [review]
Thanks for the contribution @CPL-markus!

File: modules/acm/variables.tf (hunk 1 above)
Comment (CPL-markus) [reply modules/acm/variables.tf]: I agree with you and changed it. :-)

**Review by apeabody — APPROVED** [review]
Thanks for the contribution @CPL-markus!

**Comment by apeabody** [comment]
@CPL-markus - Can you please rebase/merge? Thanks!

**Comment by CPL-markus** [comment]
@apeabody I rebased it, but merging is blocked as I need your help in running the pipeline again. :-)

**Pull request merged**